internal/page/share_code.go

```diff
@@ -68,14 +82,15 @@ func (s *ShareCodeSender) sendCertificateProvider(ctx context.Context, appData A
 		DonorFirstNames: donor.Donor.FirstNames,
 		SessionID:       appData.SessionID,
 	}); err != nil {
-		return fmt.Errorf("creating sharecode failed: %w", err)
+		return "", fmt.Errorf("creating sharecode failed: %w", err)
 	}
 
-	if _, err := s.notifyClient.SendEmail(ctx, donor.CertificateProvider.Email, email.WithShareCode(shareCode)); err != nil {
-		return fmt.Errorf("email failed: %w", err)
+	id, err := s.notifyClient.SendEmail(ctx, donor.CertificateProvider.Email, email.WithShareCode(shareCode))
```

Comment:
finally getting some use out of the first return value 💪

Reply:
We got there 😂